src/redux/actions/cellSets/createCellSet.js

```diff
@@ -78,18 +78,12 @@ const createCellSet = (experimentId, name, color, cellIdsSet) => async (dispatch
           createCellSetJsonMerger(dataForUpload, 'scratchpad'),
         ),
       },
+      false,
```

Comment:
This false is being passed as `extras` (the 3rd parameter for `fetchApi` is `extras`, an object)

Reply:
That's a very good point. What do you think it's best:
1. Add an empty object to each of the function calls that need to set the 4th parameter
`fetchAPI(URL, params, {}, false)`
2. Change `fetchApi` so that it receives just an object and can do restructuring of the parameters
`fetchAPI({url, params, returnValue: false})

Reply:
I wish we could name the optional parameters we are defining like in python :(

Reply:
well the nodejs way of doing named parameters is passing the object that's why I suggested it. I feel having to add empty parameters is much more error-prone in addition to make it easier to introduce bugs